lib/zendesk_apps_support/validations/svg.rb

```diff
@@ -4,33 +4,38 @@
 module ZendeskAppsSupport
   module Validations
     module Svg
-      @strip_declaration = Loofah::Scrubber.new do |node|
-        node.remove if node.name == 'xml' && node.children.empty?
-      end
-
-      @empty_malformed_markup = Loofah::Scrubber.new do |node|
-        node.next.remove while node.name == 'svg' && node.next
-      end
+      # whitelist elements and attributes used in Zendesk Garden assets
+      Loofah::HTML5::WhiteList::ALLOWED_ELEMENTS_WITH_LIBXML2.add 'symbol'
+      Loofah::HTML5::WhiteList::ACCEPTABLE_CSS_PROPERTIES.add 'position'
 
       # CRUFT: ignore a (very specific) style attribute which Loofah would otherwise scrub.
       # This attribute is deprecated (https://www.w3.org/TR/filter-effects/#AccessBackgroundImage)
       # but is included in many of the test apps used in fixtures for tests in ZAM, ZAT etc.
-      @remove_enable_background = Loofah::Scrubber.new do |node|
-        match_pattern = Regexp.new("enable-background:.*?(\;|\z)")
+      Loofah::HTML5::WhiteList::ACCEPTABLE_CSS_PROPERTIES.add 'enable-background'
+
+      @strip_declaration = Loofah::Scrubber.new do |node|
+        node.remove if node.name == 'xml' && node.children.empty?
+      end
+
+      @strip_spaces_between_css_attrs = Loofah::Scrubber.new do |node|
```

Comment:
Why do we do this?

Reply:
Loofah condenses these spaces in it's optimisation of the markup, so stripping them before sanitising ensures we don't unnecessarily trigger a rewrite.* 

*(This is if you're referring to the `@strip_spaces_between_css_attrs` - I think you are, but its hard to tell)

Reply:
Could you add a comment please?